docs/DeveloperGuide.adoc

```diff
@@ -725,13 +725,74 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |add a new person |
 
+|`* * *` |user |add description to my contacts |I can make my contacts feel unique
+
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |find a person by email |locate details of persons without having to go through the entire list
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |clear the storage in the address book |restart and empty the address book
+
+|`* * *` |user |find a person by phone number |locate details of persons without having to go through the entire list
```

Comment:
There are some similar find user stories, may be you can also combine them into `find a person by any person's attributes`